packages/react-sdk/src/page-meta-generator.ts

```diff
@@ -11,27 +17,33 @@ export type PageMeta = {
 };
 
 export const generatePageMeta = ({
-  scope,
+  scope: resourcesScope,
   page,
   dataSources,
 }: {
   scope: Scope;
   page: Page;
   dataSources: DataSources;
 }) => {
+  // reserve initial parameters
+  const scope = createScope(["params", "resources"]);
```

Comment:
is this initialScope and the one passed the real one? I got a bit confused  with semantic difference between passed scope and this one and why the other one is maped as resourcesScope

Reply:
if the generatePageMeta always received resourcesScope maybe its better to call it that way and not map?